Gemfile

```diff
@@ -55,6 +55,7 @@ gem 'conjur-policy-parser', path: 'gems/policy-parser'
 gem 'conjur-rack', '~> 4'
 gem 'conjur-rack-heartbeat'
 gem 'rack-rewrite'
+gem 'yajl-ruby', require: 'yajl'
```

Comment:
@cyberark/conjur-core-team need to review any new gems that are added to the project.

Reply:
@telday do you think that a comment on this gem will help developers understand its purpose?